generators/app/templates/src/app/_app.module.ts

```diff
@@ -54,7 +62,7 @@ import { AppRoutingModule } from './app-routing.module';
 <% } else if (props.ui === 'bootstrap') { -%>
     NgbModule.forRoot(),
 <% } else if (props.ui === 'ionic') { -%>
-    IonicModule.forRoot(AppComponent, { locationStrategy: 'path' }),
+    IonicModule.forRoot(AppComponent, { locationStrategy: <%- props.location  === 'hash' ? "'hash'": "'path'" %> }),
```

Comment:
Hash is the default here, so no need to be told explicitely for this use case

Reply:
I've worried about if ionic team change the default. So I prefer to not rely on its default value.